locale/en_US/manager.po

```diff
@@ -1747,3 +1747,15 @@ msgstr "The currency of the payment amount, such as USD"
 
 msgid "emailTemplate.variable.site.siteTitle"
 msgstr "Name of the website when more than one journal is hosted"
+
+msgid "mailable.validateEmailContext.name"
+msgstr "Email Validated (Journal)"
+
+msgid "mailable.validateEmailContext.description"
+msgstr "Automatic email sent to the user after registration from the journal"
+
+msgid "mailable.validateEmailSite.name"
+msgstr "Email Validate (Site)"
+
+msgid "mailable.validateEmailSite.description"
+msgstr "Automatic email sent to the user after registration from the site"
```

Comment:
The site-level messages can be adapted like the ones above, and then moved into pkp-lib.